requirements.txt

```diff
@@ -1,2 +1,2 @@
  # installs dependencies from ./setup.py, and the package itself, in editable mode
-.
+.pymongo
```

Comment:
we're keeping our dependencies in setup.py, currently line 14.
Would you mind adding it to the list there?

Reply:
will do so